app/src/main/res/layout/viewholder_news_item.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="@dimen/viewholder_news_container"
+    android:background="@color/white">
+
+    <RelativeLayout
+        android:id="@+id/vRelativeContainer"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <ImageView
```

Comment:
Instead of ImageView. We are going to use `SimpleDraweeView` instead. It allows to bring images from internet via url